src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -77,7 +78,8 @@ class ResumeCtrlTest : public ::testing::Test {
       , kTestDevId_(5u)
       , kTestGrammarId_(10)
       , kHash_("saved_hash")
-      , kAppResumingTimeout_(30000000u) {}
+      , kAppResumingTimeout_(30000000u)
```

Comment:
@LitvinenkoIra please add comment that it is milliseconds, Also please try to use smaller timeout to UT's I suppose that this ones it very long.